string Feature1UUID
string Feature2UUID
float64 Coefficient
